pkg/registry/core/node/strategy.go

```diff
@@ -127,8 +127,14 @@ type nodeStatusStrategy struct {
 var StatusStrategy = nodeStatusStrategy{Strategy}
 
 func (nodeStatusStrategy) PrepareForCreate(ctx context.Context, obj runtime.Object) {
-	_ = obj.(*api.Node)
+	node := obj.(*api.Node)
 	// Nodes allow *all* fields, including status, to be set on create.
+
+	if !utilfeature.DefaultFeatureGate.Enabled(features.DynamicKubeletConfig) {
+		node.Status.Config.Active = nil
```

Comment:
seems like the whole config field should disappear in that case

Reply:
also need to strip this field on update

Reply:
Good catch, I changed it to a pointer type midway through working on this and forgot to update this.

Reply:
updated PrepareForUpdate as well